recipes/mss/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set version = "1.1.0" %}
+
+
+package:
+  name: mss
+  version: {{ version }}
+
+source:
+    git_url: https://bitbucket.org/wxmetvis/mss.git
+    tag: 1.1.0
+
+build:
+    number: 0
+    script: python setup.py install --record record.txt
+    entry_points:
+      - mss = msui.mss_pyui:main
+
+requirements:
+  build:
+    - basemap ==1.0.7
+    - chameleon >=2.24
+    - curl ==7.26
+    - cycler ==0.10.0
+    - execnet ==1.2.0
+    - freetype >=2.5.0
+    - functools32 >=3.2.3.2
+    - geopy >=1.10.0
+    - hdf5 >=1.8.15
+    - jpeg >=8d
+    - libpng >=1.5.0
+    - lxml ==3.6.0
+    - matplotlib >=1.4.0
+    - netcdf4 >=1.0
+    - numpy ==1.10.4
+    - paste >=1.7.5
+    - pillow >=3.2.0
+    - pyqt >=4.11.4
+    - python ==2.7.11
+    - python-dateutil ==2.5.2
+    - pytz ==2016.3
+    - qt >=4.8.6
+    - requests ==2.9.1
+    - scipy >=0.17.0
+    - sip ==4.16.9
+    - six ==1.10.0
+    - tk >=8.5.18
+  run:
+    - basemap ==1.0.7
+    - chameleon >=2.24
+    - curl ==7.26
+    - cycler ==0.10.0
+    - execnet ==1.2.0
+    - freetype >=2.5.0
+    - functools32 >=3.2.3.2
+    - geopy >=1.10.0
+    - hdf5 >=1.8.15
+    - jpeg >=8d
+    - libpng >=1.5.0
+    - lxml ==3.6.0
+    - matplotlib >=1.4.0
+    - netcdf4 >=1.0
+    - numpy ==1.10.4
+    - paste >=1.7.5
+    - pillow >=3.2.0
+    - pyqt >=4.11.4
+    - python ==2.7.11
```

Comment:
To handle Python versions, we use `skip: true` with selectors in the build section. So, this would be `skip: true  # [py3k]`.

Is there a reason this was pinned down to the patch version?